appium/tests/screenobjects/base.screen.ts

```diff
@@ -6,7 +6,8 @@ const SELECTORS = {
   ERROR_HEADER: '-ios class chain:**/XCUIElementTypeStaticText[`label == "Error"`]',
   OK_BUTTON: '~OK',
   ERROR_FES_HEADER: '-ios class chain:**/XCUIElementTypeStaticText[`label == "Startup Error"`]',
-  RETRY_BUTTON: '~Retry'
+  RETRY_BUTTON: '~Retry',
+  CURRENT_ERROR: '-ios predicate string:type == "XCUIElementTypeAlert"'
```

Comment:
It'll be better to call this selector `CURRENT_MODAL`, as in some cases there won't be errors, just some app message.

Reply:
updated